### DISCUSSION TITLE
Clear previous scan results?

### DISCUSSION THREAD

**Discussion by supersky2001** [seemoo-lab/AirGuard#105] [Ideas]
How does a user clear the scan results to start a new scan?  After travelling with the app for a while it gets full of devices that I'll probably never encounter again.
A tap on "clear data" in Android settings works of course...

**Comment by imbissbudenaesthetik** [comment]
Hi,

A few versions of AirGuard ago, we implemented a feature that auto-deletes devices. You can select the timeframe for device deletion (7 days to 3 months) and whether only devices that have not been identified as potentially following you should be deleted, or all devices, including potentially tracking devices, should be deleted.
